contrib/libvirt/template.py

```diff
@@ -0,0 +1,126 @@
+pkgname = "libvirt"
+pkgver = "10.2.0"
+pkgrel = 0
+build_style = "meson"
+configure_args = [
+    "-Dattr=enabled",
+    "-Dblkid=enabled",
+    "-Dcapng=enabled",
+    "-Dcurl=enabled",
+    "-Ddriver_qemu=enabled",
+    "-Ddriver_secrets=enabled",
+    "-Dfirewalld=enabled",
+    "-Dfirewalld_zone=enabled",
+    "-Dfuse=enabled",
+    "-Dlibiscsi=enabled",
+    "-Dlibnl=enabled",
+    "-Dlibssh2=enabled",
+    "-Dno_git=true",
+    "-Dnumactl=enabled",
+    "-Dpackager=Chimera Linux",
+    "-Dpciaccess=enabled",
+    "-Dstorage_dir=enabled",
+    "-Dstorage_disk=enabled",
+    "-Dstorage_fs=enabled",
+    "-Dstorage_lvm=enabled",
+    "-Dstorage_mpath=enabled",
+    "-Dstorage_scsi=enabled",
+    "-Dstorage_zfs=enabled",
+    "-Dudev=enabled",
+    "-Dyajl=enabled",
+    f"-Dpackager_version={pkgver}",
+    "-Dqemu_user=virtqemu",
+    "-Dqemu_group=virtqemu",
+]
+hostmakedepends = [
+    "libxml2-progs",
+    "lvm2",  # buildtime check
+    "meson",
+    "mkfs",  # buildtime check
+    "mount",  # buildtime check
+    "perl",
+    "pkgconf",
+    "python-docutils",
+    "xsltproc",
+]
+makedepends = [
+    "acl-devel",
+    "attr-devel",
+    "bash-completion",
+    "device-mapper-devel",
+    "fuse-devel",
+    "glib-devel",
+    "gnutls-devel",
+    "libcap-ng-devel",
+    "libcurl-devel",
+    "libiscsi-devel",
+    "libnl-devel",
+    "libnuma-devel",
+    "libpcap-devel",
+    "libpciaccess-devel",
+    "libsasl-devel",
+    "libssh-devel",
+    "libssh2-devel",
+    "libtirpc-devel",
+    "libxml2-devel",
+    "linux-headers",
+    "parted-devel",
+    "polkit-devel",
+    "readline-devel",
+    "udev-devel",
+    "yajl-devel",
+]
+checkdepends = [
+    "pahole",
+    "python-black",
+    "python-flake8",
+    "python-pytest",
+]
+depends = [
```

Comment:
if services request dbus names this should depend on dbus too

Reply:
i do not know how to figure that out, I'll leave it as is for now